packages/itmat-interface/test/serverTests/file.test.ts

```diff
@@ -35,6 +36,8 @@ if (global.hasMinio) {
     });
 
     beforeAll(async () => { // eslint-disable-line no-undef
+        /* Choose API version used in this test file */
+        apiPath = '/api/'.concat(config.apiVersions[config.apiVersions.length - 1]);
```

Comment:
When testing we still need to verify the previous versions are still passing there tests, otherwise this might lead us into error when older software uses them and they break everything.